info.json

```diff
@@ -60,6 +60,11 @@
                 "form"
             ]
         },
+        "views": {
+            "navigation": [
+                "Resources"
+            ]
+        },
         "roles": [
             {
                 "name": "Full App Permissions"
```

Comment:
Do we really need full app permission?

Reply:
Yes, we have provided selective permission on the "Employee Watchlist" and "Sensitive Files" modules under 'Full app permission role'.